mvp-spec/operator-api.md

```diff
@@ -16,13 +16,13 @@ These endpoints need to support both the "3PC" and "no 3PC" contexts.
 
 In practice, this will translate into endpoints available under different root paths. Example paths are specified in the last column of the table.
 
-| Endpoint                                | Input*                  | Output                                                | Description                                                                                                                                                                               | Notes                                                                                                                                                                                                                                                                                   | Rest                        | Redirect                        |
-| --------------------------------------- | ----------------------- | ----------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | --------------------------- | ------------------------------- |
-| Read data                               | -                       | List of IDs (if any).<br>List of preferences (if any) | If ID 🍪 exist, return it.<br>If preferences 🍪 exist, return them                                                                                                                        | -                                                                                                                                                                                                                                                                                       | GET /v1/json/read           | GET /v1/redirect/read           |
```

Comment:
I am ok for the cookies emoticons in the schemas but I am more skeptical about in the written documentation. What about using plain English ?